docs/docs/examples/model-inference/Huggingface-Model-Inference/index.md

```diff
@@ -15,25 +15,22 @@ Dolly 2.0 package is open source, including the training code, dataset, and mode
 
 ## Running locally
 ### Prerequisites
-- A NVIDIA GPU
-- Python
+1. A NVIDIA GPU
+1. Python
 
 ### Installing dependencies
 
 
 
 ```bash
-%%bash
 pip -q install git+https://github.com/huggingface/transformers # need to install from github
-pip -q install accelerate>=0.12.0
+pip -q install accelerate # ensure you are using version higher than 0.12.0
```

Comment:
The previous version of this line would have updated any old versions of `accelerate` if they were lower than 12.  The new version of this line puts the onus on the user to know what version they may have installed already. And the way they can be sure they have a version higher than 0.12.0 is to add `>=0.12.0` to the end of the line.

Perhaps we can just add the comment to the end of the previous line?  

```suggestion
pip -q install accelerate>=0.12.0 # ensure you are using version higher than 0.12.0
```

Reply:
Added --upgrade flag. It will get latest version if any other is already installed. Current version is 0.27.2 so that shouldn't cause any issues